Laba3/CreateCalendar.php

```diff
@@ -0,0 +1,159 @@
+<?php
+	
+	define('MIN_YEAR', 1);
+	define('MAX_YEAR', 2049);
+	define('MIN_COURSE', 1);
+	define('MAX_COURSE', 3);
+	
+	if (isset($_POST['year'])&& isset($_POST['course'])) {
```

Comment:
Следите за код стайлом
```suggestion
	if (isset($_POST['year']) && isset($_POST['course'])) {
```